engine/apps/api/serializers/user.py

```diff
@@ -270,14 +270,19 @@ class Meta:
         ]
 
 
-class UserLongSerializer(UserSerializer):
+class UserIsCurrentlyOnCallSerializer(UserShortSerializer, EagerLoadingMixin):
     context: UserSerializerContext
 
     teams = FastTeamSerializer(read_only=True, many=True)
     is_currently_oncall = serializers.SerializerMethodField()
 
-    class Meta(UserSerializer.Meta):
-        fields = UserSerializer.Meta.fields + [
+    SELECT_RELATED = ["organization"]
+    PREFETCH_RELATED = ["teams"]
```

Comment:
this + switching from the base class of `UserSerializer` to `UserShortSerializer` results in wayyyy less SQL queries being needed (verified via django-silk)